libcontainer/cgroups/fs2/fs2.go

```diff
@@ -13,6 +13,10 @@ import (
 
 type parseError = fscommon.ParseError
 
+func malformedLine(path, file, line string) error {
```

Comment:
You copied the function over, without copying its docstring.

Perhaps it's better to move it to fscommon and make it available for all cgroups code?

Reply:
If you do something like this, please make it as a separate commit.

Reply:
Got it.